src/routes/artist.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const artistController = require('../controllers/artist');
+
+const router = express.Router();
+
+router.post('/', artistController.createArtist);
```

Comment:
As above I would group these together. Whether you want to group them by the common object (`router`) or by verb is up to you, though whatever approach you choose should remain consistent throughout the project